### PR TITLE
sample data: move screenshot image up

### PR DESCRIPTION
**Pull request by ninsbl**
Following up #76

@lucadelu, I could not find your review either. From your comments I understand you prefer having the screenshot where it is mentioned. I agree on your point and moved it up in this PR.

Is it along the lines you thought?

Did you have more comments?

### CODE REVIEW

File: content/download/data.en.md [hunk 1]
```diff
@@ -12,15 +12,17 @@ layout: "data"
 
 [ [**North Carolina dataset**](#NorthCarolinaDataset) | [**Spearfish dataset**](#SpearFishDataset) | [**Datasets for special topics**](#SpecialTopicDatasets) | [**Global datasets**](#GlobalDatasets) ]
 
-<div class="alert rounded-0 alert-default">
+<img src="../../images/gallery/gui/grass740_startup_sample_data_download_arrow.png" width="60%" alt="" style="float:right">
+
+<table><tr><th>
+<div class="alert rounded-0 alert-default" width="40%">
```
Comment (lucadelu) [inline content/download/data.en.md:18]:
```suggestion
```

File: content/download/data.en.md [hunk 2]
```diff
@@ -12,15 +12,17 @@ layout: "data"
...
 <i class="fa fa-arrow-right"></i> This section gathers sample datasets for learning or testing GRASS GIS. Datasets are made available as ready-to-use GRASS GIS <b>locations</b> and sample data in other
 common GIS formats.
-
-</div>
-
-<div class="alert rounded-0 alert-default">
+<br><br>
```
Comment (lucadelu) [inline content/download/data.en.md:21]:
```suggestion
```

File: content/download/data.en.md [hunk 3]
```diff
@@ -12,15 +12,17 @@ layout: "data"
...
 <i class="fa fa-arrow-right"></i> Note: an easy way to download sample data is from the <a href="#DownloadGUI">download button on the GRASS GIS startup screen</a>.
 </div>
+</th><th>
+</th></tr></table>
```
Comment (lucadelu) [inline content/download/data.en.md:25]:
```suggestion
```

File: content/download/data.en.md [hunk 4]
```diff
@@ -12,15 +12,17 @@ layout: "data"
...
+<table><tr><th>
+<div class="alert rounded-0 alert-default" width="40%">
 <i class="fa fa-arrow-right"></i> This section gathers sample datasets for learning or testing GRASS GIS. Datasets are made available as ready-to-use GRASS GIS <b>locations</b> and sample data in other
 common GIS formats.
-
-</div>
-
-<div class="alert rounded-0 alert-default">
+<br><br>
 <i class="fa fa-arrow-right"></i> Note: an easy way to download sample data is from the <a href="#DownloadGUI">download button on the GRASS GIS startup screen</a>.
```
Comment (lucadelu) [inline content/download/data.en.md:22]:
```suggestion
<i class="fa fa-arrow-right"></i> Note: an easy way to download sample data is from the <a href="#DownloadGUI">download button on the GRASS GIS startup screen</a>.
<img src="../../images/gallery/gui/grass740_startup_sample_data_download_arrow.png" width="60%" alt="" style="float:right">
```

**Comment by lucadelu** [comment]
> If I put the image in the
> section, the grey background covers 100% of the available width, which looks a bit odd, cause the image has white background.
> If the image is placed above the
> , the grey background is displayed on top of the image.
> 
> That is why I added the table...
> 
> Any idea how to limit the grey background of the
> section to the remaining proportion of the width without the table?

I like the idea to see it in the gray box since is related to that section, however you can add the picture in a div out of alert (maybe you can add a class="row")

File: content/download/data.en.md (hunk 4 above)
Comment (neteler) [reply content/download/data.en.md:22]: A question: elsewhere we use absolute path to gallery... what is prefered?

**Comment by lucadelu** [comment]
ok, if we want to follow this way, maybe it is better to use row bootstrap class, please submit and I'll propose enhancement

**Pull request merged**